src/App.js

```diff
@@ -45,8 +57,8 @@ const App = () => {
       <h1>{title}</h1>
       <AddTodoForm onAddTodo={addTodo}/>
       {/* Pass `setNewTodo` as a callback handler prop named `onAddTodo` to the `AddTodoForm` component */}
-      <p>{}</p>
-      <TodoList todoList={todoList} onRemoveTodo={removeTodo}/>
+      <p>{isLoading ? "Loading..." : (<TodoList todoList={todoList} onRemoveTodo={removeTodo}/>)}</p>
```

Comment:
The inner ternary is almost correct but you don't need to wrap the whole thing with a <p> tag. You can just wrap the "Loading..." text in a <p> tag. This is causing a validateDomNesting error in the console.